### DISCUSSION TITLE
使用vc-drawings绘制图形回显和编辑问题

### DISCUSSION THREAD

**Discussion by coderyx-dev** [zouyaoji/vue-cesium#597] [Q&A]
使用vc-drawings绘制图形之后使用vc-datasource-custom存储并回显，但是此时的图形就不能编辑了，如何解决这个问题

**Comment by heroqyc** [comment]
你好想问一下  绘制的矩形圆形这种是怎么回显的？renderDatas里面只有两个坐标点

**Reply by PetukhovArt** [reply]
this graphics has only two points to render, as i searched , for example circle must have center and second point , that helps to count radius i think.

**Comment by NanF007** [comment]
@coderyx-dev 
使用vc-drawings绘制图形之后使用vc-datasource-custom存储并回显，但是此时的图形就不能编辑了，如何解决这个问题
大佬！这个问题有答案了吗
